server/accounts.go

```diff
@@ -70,6 +71,8 @@ type Account struct {
 	lds         string  // loop detection subject for leaf nodes
 	siReply     []byte  // service reply prefix, will form wildcard subscription.
 	prand       *rand.Rand
+	eventids    *nuid.NUID
+	eventidsmu  sync.Mutex
```

Comment:
adding this mu rather than relying on the account one solved the race conditions, boggle.